js_modules/dagit/packages/ui/src/components/NonIdealState.tsx

```diff
@@ -42,14 +42,16 @@ export const NonIdealState: React.FC<NonIdealStateProps> = ({
         <Spinner purpose="section" />
       ) : icon === 'no-results' ? (
         <Icon name="search" size={48} color={Colors.Gray400} />
-      ) : (
+      ) : icon ? (
         <Icon name={icon} size={48} color={Colors.Gray400} />
-      )}
+      ) : null}
```

Comment:
Why do we need a no-icon state?

Reply:
The icon takes up too much space when shown in smaller spaces. In this case the right panel.

Reply:
Hmm, maybe we need a smaller NonIdealState.